store/tikv/coprocessor.go

```diff
@@ -497,12 +521,28 @@ func (worker *copIteratorWorker) run(ctx context.Context) {
 		if respCh == nil {
 			respCh = task.respChan
 		}
-
 		worker.handleTask(ctx, task, respCh)
 		close(task.respChan)
-		if worker.respChan != nil {
+
+		worker.actionOnExceed.cond.L.Lock()
+		if task.id > worker.actionOnExceed.cond.maxRunningTaskID {
+			worker.actionOnExceed.cond.maxRunningTaskID = task.id
+		}
+		// If actionOnExceed has been triggered and there is no ticket have been spilled before,
+		// spill one ticket.
+		if worker.actionOnExceed.cond.exceed && !worker.actionOnExceed.cond.teared {
```

Comment:
`cond.teared` is set to `false` only the first time Action is triggered?

If so, we can only tear off one token?

Reply:
Exactly, we only tear one ticket each time oom action triggered currently.